.github/workflows/release-notification.yml

```diff
@@ -3,7 +3,7 @@ name: Release notification
 on:
   push:
     tags:
-      - dialtone/v*
+      - 'dialtone/v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Is this regex? If so you'll need to escape the dot right?

Reply:
It is glob pattern, the reference link with the examples is at the bottom of the PR description.

https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

Reply:
Alright, yeah it's fine then